server.js

```diff
@@ -3,16 +3,21 @@ var express = require('express');
 var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
+const port = 5000;
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
-
+var reactClient = require('./routes/reactClient')
+var cors = require('cors');
 var app = express();
 
+//set port 5000
+app.listen(port, () => console.log(`Listening on port ${port}`));
```

Comment:
This can be removed regarding the reference to bin/www comment

Reply:
I tried to remove it and it failed to start the server